juju/juju-gui
# Add error boundaries to Profile components

## Done 

- Added error boundaries to Profile section of the GUI to improve resilience.

## QA

- Run this feature branch
- Click each link in the side nav of Profile to ensure each view loads
- Open `credential-list.js` and add `throw new Error('Error!')` to the last line of the `Constructor()` function.
- Reload the GUI and you should see an error message where the Credentials table should be.

## Demo

Before - Without error boundaries: https://cl.ly/8788d419815b
After - With error boundaries: https://cl.ly/9fd93f8d66ba

Fixes #3865
